package.json

```diff
@@ -16,7 +16,6 @@
     "clean": "shx rm -rf ./bundles ./docs ./lib ./lib-esm ./coverage",
     "clean:all": "npm run clean && shx rm -rf ./node_modules ./package-lock.json && git clean -fX -e \"!*.pem\"",
     "docs": "typedoc",
-    "docs:watch": "watch \"npm run docs\" ./src",
```

Comment:
Removed it as it was never used and required a package that had a vulnerability in it.